contrib/smtp_proxy/filters/network/source/smtp_session.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#include <cstdint>
+
+#include "source/common/common/logger.h"
+
+#include "contrib/smtp_proxy/filters/network/source/smtp_transaction.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace SmtpProxy {
+
+class SmtpSession {
+public:
+  enum class State {
+    CONNECTION_REQUEST = 0,
+    CONNECTION_SUCCESS = 1,
+    SESSION_INIT_REQUEST = 2,
+    SESSION_IN_PROGRESS = 3,
+    SESSION_TERMINATION_REQUEST = 4,
+    SESSION_TERMINATED = 5,
+    UPSTREAM_TLS_NEGOTIATION = 6,
+    DOWNSTREAM_TLS_NEGOTIATION = 7,
+    SESSION_AUTH_REQUEST = 8,
+  };
+
+  void setState(SmtpSession::State state) { state_ = state; }
+  SmtpSession::State getState() { return state_; }
+
+  void SetTransactionState(SmtpTransaction::State state) { smtp_transaction_.setState(state); };
```

Comment:
Change to _setTransactionState_. Function names should start with small letter.

Reply:
Done